stable/ecr-cleanup/templates/deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
             - --v={{ .Values.controller.verbosity }}
             - --interval={{ .Values.controller.interval }}
             - --registry-id={{ .Values.controller.registryID }}
+{{- if .Values.controller.image_whitelist }}
```

Comment:
I might be overthinking it a bit but `whitelist` feels misleading since it's a list of things that shouldn't be cleaned up. What would you think about renaming it to `keep_filters` to match the underlying argument?

Reply:
👍 sounds good to me.  Updated to `keep_filters`.